Neos.ContentRepository/Classes/Domain/Context/DimensionSpace/Repository/InterDimensionalFallbackGraph.php

```diff
@@ -216,33 +216,60 @@ public function getSubgraphByDimensionSpacePoint(Domain\ValueObject\DimensionSpa
     /**
      * @param Domain\ValueObject\DimensionSpacePoint $origin
      * @param bool $includeOrigin
+     * @param Domain\ValueObject\DimensionSpacePointSet|null $excludedSet
      * @return Domain\ValueObject\DimensionSpacePointSet
      * @throws DimensionSpacePointNotFound
      * @throws DimensionSpace\Exception\FallbackInitializationException
      */
-    public function getSpecializationSet(Domain\ValueObject\DimensionSpacePoint $origin, bool $includeOrigin = true): Domain\ValueObject\DimensionSpacePointSet
-    {
+    public function getSpecializationSet(
+        Domain\ValueObject\DimensionSpacePoint $origin,
+        bool $includeOrigin = true,
+        Domain\ValueObject\DimensionSpacePointSet $excludedSet = null
```

Comment:
Couldn't we just get the whole set and then do `A - B` as a operation on a `DimensionSpacePointSet` (which returns a new set ofc)?